src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/GenericComponent.tsx

```diff
@@ -129,14 +109,15 @@ export function GenericComponent(props: IGenericComponentProps) {
     return null;
   }
 
-  const RenderComponent = components.find((component: any) => component.name === props.type).Tag;
+  const RenderComponent = components.find((componentCandidate: any) => componentCandidate.name === props.type).Tag;
 
   const RenderLabel = () => {
     const labelText = getTextResource(props.textResourceBindings.title, textResources);
+    const helpText = getTextResource(props.textResourceBindings.help, textResources);
```

Comment:
Do we need a null check here as well, like on l.147?

Reply:
I guess that would only be a problem if the component was missing the `textResourceBidning` 
 in `formlayout.json`. No harm in adding the check i guess. Fixing.